x-pack/plugins/alerting/server/routes/rule/apis/find/find_rules_route.ts

```diff
@@ -102,6 +102,7 @@ const buildFindRulesRoute = ({
     router.post(
       {
         path,
+        options: { access: 'internal' },
```

Comment:
In this file, `GET` method has `public route and path` at line 45  and `POST` method uses `internal path` at line 101. However in the list internal path is listed as `GET`:
GET /internal/alerting/rules/_find | internal | ❌ |  
-- | -- | -- | --

Reply:
Related PR with a fix: https://github.com/elastic/kibana/pull/193757